_posts/2020-03-13-Azure-Functions-Portal-Preview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Announcing public preview for Azure Functions portal UX"
+tags: 
+    - Function
+    - Preview
+    - UX
+author_name: "Byron Tardif"
+---
+
+## Access the preview
+
+The new user experience for **Azure Functions** is now available for all users in the **Azure Portal**.
+
+You can try the new experience by clicking on the "*Preview the new Azure Functions management experience*" banner in your function apps.
+
+![Access the Functions Preview UX]({{ site.baseurl }}/media/2020/03/functions_preview.gif)
+
+## Function App Overview
+
+Once you are in the new experience, you will notice that the new  **Azure Functions** UX is now consistent with the rest of the **Azure Portal**.
+
+This UX is powered by the same [**Azure Functions Runtime API**](https://docs.microsoft.com/azure/azure-functions/functions-runtime-overview) except they are now exposed through [**ARM**](https://azure.microsoft.com/features/resource-manager/). This allows for better caching through the [**Azure Resource Graph**](https://azure.microsoft.com/features/resource-graph/), as well as support for [**RBAC**](https://docs.microsoft.com/azure/role-based-access-control/overview), and enables new scenarios like being able to manage Azure Functions hosted on an [**Internal Load Balancer App Service Environment ASE**](https://docs.microsoft.com/azure/app-service/environment/create-ilb-ase).
```

Comment:
We're not technically supporting ARG just yet so it's not entirely accurate.  We've also had some snags in discussion in getting Paul to support it so most likely we won't have it any time soon.

Reply:
Instead of ARG, I would say that we've implemented our own caching layer which improves API and UX performance or something which is more accurate.

Reply:
ok, I'll push an edit for this

Reply:
I'll just say "...this allows for better caching, as well as ..."